src/jquery-data-remote.js

```diff
@@ -47,7 +47,7 @@
         if (options.handlebars === true || $target.find('[type="text/x-handlebars-template"]').length) {
           if (options.handlebarsTemplate) {
             $handlebarsTemplate = $(options.handlebarsTemplate);
-          } else 
+          } else {
```

Comment:
you can actually delete lines 48-50. it's not an actual if statement thats needed, cause the if statement is happening on lines 51-53